src/Renderer/c3DEngine.js

```diff
@@ -18,6 +18,9 @@ function c3DEngine(rendererOrDiv, options = {}) {
     if (options.alpha === undefined) {
         options.alpha = true;
     }
+    if (options.alphaClearValue === undefined) {
```

Comment:
Beware: `c3DEngine` constructor receive `options.renderer` object as `options`.

In this PR you sometimes write in `options.renderer.alphaClearValue` (like now) and sometimes in `options.alphaClearValue` (like in `GlobeView.preRender`).

Last but not least: I *think* we should reserve `options` for fixed options = stuff that you can't change dynamically.
For this `alphaClearValue` I *believe* you maybe doesn't even need this option.
Here's an alternative implementation:
-  this.renderer.setClearColor(0x030508, 0); // <- do this once when creating the miniglobe
- change `setClearColor`calls to: `this.renderer.setClearColor(0x030508, renderer.getClearAlpha()); `